packages/contracts-bedrock/src/universal/FeeVault.sol

```diff
@@ -76,7 +76,10 @@ abstract contract FeeVault {
         emit Withdrawal(value, RECIPIENT, msg.sender, WITHDRAWAL_NETWORK);
 
         if (WITHDRAWAL_NETWORK == WithdrawalNetwork.L2) {
-            SafeCall.send(RECIPIENT, gasleft(), value);
+            require(
+                SafeCall.send(RECIPIENT, gasleft(), value),
+                "FeeVault: failed to send ETH to L2"
```

Comment:
nit: could we change this to `FeeVault: failed to send ETH to L2 fee recipient` to make what failed clearer?